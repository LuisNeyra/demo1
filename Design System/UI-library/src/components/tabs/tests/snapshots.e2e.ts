import { snapIt } from '../../../test';

const component = 'tabs';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap(
      'renders correctly with no props',
      `<blaze-tabs>
      <blaze-tab>This is tab one</blaze-tab>
      <blaze-tab>This is tab two</blaze-tab>
    </blaze-tabs>`
    );
    snap(
      'renders correctly with props set',
      `<blaze-tabs>
      <blaze-tab header="Tab one" open>This is tab one</blaze-tab>
      <blaze-tab header="Tab two" disabled>This is tab two</blaze-tab>
    </blaze-tabs>`
    );
  });
});
