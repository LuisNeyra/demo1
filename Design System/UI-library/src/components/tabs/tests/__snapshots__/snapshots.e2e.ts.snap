// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tabs renders renders correctly with no props 1`] = `"<blaze-tabs class=\\"hydrated\\"><!----><div class=\\"c-tabs\\"><div role=\\"tablist\\" class=\\"c-tabs\\"><div class=\\"c-tabs__nav\\"><div class=\\"c-tabs__headings\\"><button role=\\"tab\\" class=\\"c-tab-heading\\"></button><button role=\\"tab\\" class=\\"c-tab-heading\\"></button></div></div> <blaze-tab class=\\"hydrated\\"><!----><div role=\\"tabpanel\\" hidden=\\"\\" class=\\"c-tabs__tab\\">This is tab one</div></blaze-tab> <blaze-tab class=\\"hydrated\\"><!----><div role=\\"tabpanel\\" hidden=\\"\\" class=\\"c-tabs__tab\\">This is tab two</div></blaze-tab> </div></div></blaze-tabs>"`;

exports[`tabs renders renders correctly with props set 1`] = `"<blaze-tabs class=\\"hydrated\\"><!----><div class=\\"c-tabs\\"><div role=\\"tablist\\" class=\\"c-tabs\\"><div class=\\"c-tabs__nav\\"><div class=\\"c-tabs__headings\\"><button role=\\"tab\\" class=\\"c-tab-heading c-tab-heading--active\\">Tab one</button><button role=\\"tab\\" disabled=\\"\\" class=\\"c-tab-heading\\">Tab two</button></div></div> <blaze-tab header=\\"Tab one\\" open=\\"\\" class=\\"hydrated\\"><!----><div role=\\"tabpanel\\" class=\\"c-tabs__tab\\">This is tab one</div></blaze-tab> <blaze-tab header=\\"Tab two\\" disabled=\\"\\" class=\\"hydrated\\"><!----><div role=\\"tabpanel\\" hidden=\\"\\" class=\\"c-tabs__tab\\">This is tab two</div></blaze-tab> </div></div></blaze-tabs>"`;
